mlos_bench/mlos_bench/optimizer/null_optimizer.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+#
+"""
+Null optimizer for mlos_bench that proposes a single configuration.
+"""
+
+import logging
+from typing import Dict, Optional, Any
+
+from mlos_bench.service.base_service import Service
+from mlos_bench.service.types.config_loader_type import SupportsConfigLoading
+from mlos_bench.tunables.tunable_groups import TunableGroups
+from mlos_bench.optimizer.mock_optimizer import MockOptimizer
+
+_LOG = logging.getLogger(__name__)
+
+
+class NullOptimizer(MockOptimizer):
+    """
+    Null optimizer that proposes a single configuration and converges.
+    """
+
+    def __init__(self, tunables: TunableGroups,
+                 service: Optional[Service], config: Dict[str, Any]):
+        super().__init__(tunables, service, config)
+        if self._service is not None and isinstance(self._service, SupportsConfigLoading):
```

Comment:
I think inside `BaseService` there should be a `self._config_loader_service` instance that you can use for this.

Reply:
It was so common that I think I added that check and attribute alias to the base class so that we didn't have to have that `isinstance` check everywhere.

Reply:
oh, that's right. will fix it now